apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -387,7 +389,7 @@ function handleUpdateImageBlocks( calypsoPort ) {
 			return;
 		}
 		const image = get( message, 'data.payload' );
-		updateImageBlocks( select( 'core/editor' ).getBlocks(), image );
+		updateImageBlocks( select( blockEditorStore ).getBlocks(), image );
```

Comment:
Did we mean `editorStore`?

```suggestion
		updateImageBlocks( select( editorStore ).getBlocks(), image );
```